examples/java_postgresql_customvar/README.md

```diff
@@ -107,6 +107,55 @@ and install a `stable` version.
 
 This makes the `Database` custom resource available, that the application developer will use later.
 
+### Update the Database CRD
+
+Annotations are created to describe the values that should be collected and made available by the Service Binding Request's intermediary secret.
+ 
+
+```
+oc get crd databases.postgresql.baiju.dev -o yaml
+```
+ 
+The database CRD contains the list of annotations:
+
+```
+metadata:
+  annotations:
+    servicebindingoperator.redhat.io/spec.dbName: binding:env:attribute
+    servicebindingoperator.redhat.io/status.dbConfigMap-db.host: binding:env:object:configmap
+    servicebindingoperator.redhat.io/status.dbConfigMap-db.name: binding:env:object:configmap
+    servicebindingoperator.redhat.io/status.dbConfigMap-db.password: binding:env:object:configmap
+    servicebindingoperator.redhat.io/status.dbConfigMap-db.port: binding:env:object:configmap
+    servicebindingoperator.redhat.io/status.dbConfigMap-db.user: binding:env:object:configmap
+    servicebindingoperator.redhat.io/status.dbConnectionIP: binding:env:attribute
+    servicebindingoperator.redhat.io/status.dbConnectionPort: binding:env:attribute
+    servicebindingoperator.redhat.io/status.dbCredentials-password: binding:env:object:secret
+    servicebindingoperator.redhat.io/status.dbCredentials-user: binding:env:object:secret
+    servicebindingoperator.redhat.io/status.dbName: binding:env:attribute
+
+```
+
+We can add more annotations to collect values from different kinds of data structures.  
+
+To edit the CRD run the command:
+
+```
+oc edit crd databases.postgresql.baiju.dev
+```
+
+Add these annotations under `metadata.annotations` along with other annotations.
+
+```
+servicebindingoperator.redhat.io/spec.tags: binding:env:attribute
+servicebindingoperator.redhat.io/spec.userLabels.archive: binding:env:attribute
+servicebindingoperator.redhat.io/spec.secretName: binding:env:attribute
+```
+
+These annotations refer to data which can be either a number, string, boolean, or an object or a slice of arbitrary values. In the case of this example, 
+- `spec.tags` represents a sequence
+- `spec.userLabels` represents a mapping
```

Comment:
the annotation above refers `spec.userLabels.archive` - should we update this line?

Reply:
No.

`spec.userLabels` is a mapping. 
`spec.userLabels.archive` is a key.